lib/vmdb/loggers.rb

```diff
@@ -73,6 +81,7 @@ def self.create_loggers
     def self.create_multicast_logger(log_file_path, logger_class = VMDBLogger)
       logger_class.new(log_file_path).tap do |logger|
         logger.extend(ActiveSupport::Logger.broadcast($container_log)) if ENV["CONTAINER"]
+        logger.extend(ActiveSupport::Logger.broadcast($journald_log))  if $journald_log
```

Comment:
Do we need to write logs to files anymore if we have Journald?

Reply:
I figured until we integrated gathering journald in things like support logs it was a good idea to keep using the existing log files.